### PR TITLE
Add informatione about FoP User groups

### PR DESCRIPTION
**Pull request by okom3pom**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Add Friend of Presta 
| Fixed ticket | @coffrep

### CODE REVIEW

File: content/user-groups/_index.md [hunk 1]
```diff
@@ -40,3 +40,16 @@ In order to edit them, you will need a vector graphic software like Inkscape (fr
 When you have created you user group logo and published it on your forum, group, chat, or your favourite communication channel, please tell the world about it: add it to the list in the dedicated document (coming soon).
 
 This way, interested people will be able to find your group easily and join it with a few clicks.
+
+## Official user groups
+
+![Friend of Presta](fop_logotype-horizontal-blanc-avec-fond.svg)
+Friends of Presta is a French community that unites developers and integrators who work with the open-source e-commerce platform PrestaShop.
+
+Every year, Friend of Presta (FoP) organizes a day of conferences and workshops that bring together developers, integrators, and members of the PrestaShop open-source team.
```
Comment (PrestaEdit) [inline content/user-groups/_index.md:49]:
```suggestion
Every year, Friends of Presta (FoP) organizes a day of conferences and workshops that bring together developers, integrators, and members of the PrestaShop open-source team.
```

File: content/user-groups/_index.md [hunk 2]
```diff
@@ -40,3 +40,16 @@ In order to edit them, you will need a vector graphic software like Inkscape (fr
 When you have created you user group logo and published it on your forum, group, chat, or your favourite communication channel, please tell the world about it: add it to the list in the dedicated document (coming soon).
 
 This way, interested people will be able to find your group easily and join it with a few clicks.
+
+## Official user groups
+
+![Friend of Presta](fop_logotype-horizontal-blanc-avec-fond.svg)
```
Comment (PrestaEdit) [inline content/user-groups/_index.md:46]:
```suggestion
![Friends of Presta](fop_logotype-horizontal-blanc-avec-fond.svg)
```

File: content/user-groups/_index.md [hunk 3]
```diff
@@ -40,3 +40,16 @@ In order to edit them, you will need a vector graphic software like Inkscape (fr
...
 This way, interested people will be able to find your group easily and join it with a few clicks.
+
+## Official user groups
+
+![Friends of Presta](fop_logotype-horizontal-blanc-avec-fond.svg)
+Friends of Presta is a French community that unites developers and integrators who work with the open-source e-commerce platform PrestaShop.
+
+Every year, Friends of Presta (FoP) organizes a day of conferences and workshops that bring together developers, integrators, and members of the PrestaShop open-source team.
+
+FoP manages a Slack channel and a Facebook page that has attracted thousands of members who share a passion for PrestaShop
```
Comment (coffrep) [inline content/user-groups/_index.md:51]:
```suggestion
FoP manages a Slack channel and a Facebook page, attracting thousands of members who share a passion for PrestaShop.
```

File: content/user-groups/_index.md [hunk 4]
```diff
@@ -40,3 +40,16 @@ In order to edit them, you will need a vector graphic software like Inkscape (fr
...
+
+You can also join our security team for a safer e-commerce (more than 50 flaws reported on third party modules). And many projects on our [Github](https://github.com/friends-of-presta) including the [FoP Console](https://github.com/friends-of-presta/fop_console).
```
Comment (coffrep) [inline content/user-groups/_index.md:53]:
```suggestion
You can also join the organization's security team for a safer e-commerce (more than 50 flaws reported on third party modules). And many projects on [GitHub](https://github.com/friends-of-presta) including the [FoP Console](https://github.com/friends-of-presta/fop_console).
```

File: content/user-groups/_index.md [hunk 5]
```diff
@@ -40,3 +40,16 @@ In order to edit them, you will need a vector graphic software like Inkscape (fr
 When you have created you user group logo and published it on your forum, group, chat, or your favourite communication channel, please tell the world about it: add it to the list in the dedicated document (coming soon).
 
 This way, interested people will be able to find your group easily and join it with a few clicks.
+
+## Official user groups
+
+![Friends of Presta](fop_logotype-horizontal-blanc-avec-fond.svg)
+Friends of Presta is a French community that unites developers and integrators who work with the open-source e-commerce platform PrestaShop.
```
Comment (coffrep) [inline content/user-groups/_index.md:47]:
```suggestion
Friends of Presta is a French community that connects developers and integrators who work with the open-source e-commerce platform PrestaShop.
```

File: content/user-groups/_index.md [hunk 6]
```diff
@@ -40,3 +40,16 @@ In order to edit them, you will need a vector graphic software like Inkscape (fr
...
+
+Every year, Friends of Presta (FoP) organizes a day of conferences and workshops that bring together developers, integrators, and members of the PrestaShop open-source team.
+
+FoP manages a Slack channel and a Facebook page that has attracted thousands of members who share a passion for PrestaShop
+
+You can also join our security team for a safer e-commerce (more than 50 flaws reported on third party modules). And many projects on our [Github](https://github.com/friends-of-presta) including the [FoP Console](https://github.com/friends-of-presta/fop_console).
+
+[Join us to become a part of this community and help us improve PrestaShop!](https://friendsofpresta.org/en)
```
Comment (coffrep) [inline content/user-groups/_index.md:55]:
```suggestion
[Join FoP to become a part of the community and help improve PrestaShop!](https://friendsofpresta.org/en)
```

**Review by coffrep — CHANGES_REQUESTED** [review]
Hi - thanks for adding FoP to this page. I just added a few suggestions, do let me know of your thoughts!

**Review by coffrep — APPROVED** [review]

**Review by PrestaEdit — APPROVED** [review]

**Review by mflasquin — APPROVED** [review]

**Pull request merged**